improver_tests/psychrometric_calculations/condensation_trails/test_ContrailHeightExtractor.py

```diff
@@ -0,0 +1,309 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the ContrailHeightExtractor plugin"""
+
+import iris
+import numpy as np
+import pytest
+
```

Comment:
```suggestion
from typing import List, Optional, Tuple

```
Imports required for a separate change. The `typing` module is generally useful for adding type hints to function signatures.